backend/app/Services/Engineering/EngineeringDocumentService.php

```
@@ -0,0 +1,148 @@
+        $this->assertContentMatchesType($packageType, $file);
+
+        $checksum = hash_file('sha256', $file->getRealPath());
+
+        // Non-predictable, entity-scoped path on the private disk.
+        $dir = "engineering/{$meta['entity_type']}/{$meta['entity_id']}";
+        $name = Str::uuid()->toString().'.'.$ext;
+        $path = "{$dir}/{$name}";
+
+        Storage::disk($this->disk())->putFileAs($dir, $file, $name);
+
+        $doc = EngineeringDocument::create([
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\EngineeringDocument::create().


(staticMethod.notFound)